aten/src/ATen/mps/MPSHooks.mm

```diff
@@ -19,7 +19,16 @@
   return at::mps::is_available();
 }
 
-bool MPSHooks::isOnMacOS13orNewer(unsigned minor) const {
+bool MPSHooks::isOnMacOSorNewer(unsigned major, unsigned minor) const {
+  if (major >= 14) {
+    if (major > 14) {
+      TORCH_WARN("Can't check whether running on ", major, ".", minor, "+ returning one for 14.0+");
+    } else if (minor > 0) {
+      TORCH_WARN("Can't check whether running on 14.", minor, "+ returning one for 14.0+");
+    }
+    return is_macos_13_or_newer(MacOSVersion::MACOS_VER_14_0_PLUS);
```

Comment:
Should is_macos_13_or_newer be renamed as well?

Reply:
This is part of the stable API, and `is_macos_13_or_newer(MACOS_VER_14_0_PLUS)` sounds like a reasonable API call, but sure, I can rename it